### ISSUE TITLE
make syncdb fails

### ISSUE THREAD

**Issue by gjost** [densho/ddr-local#224]
Install fails at the `sudo make syncdb` step.
![make_syncdb](https://user-images.githubusercontent.com/38606/34064694-12845cc8-e1b0-11e7-85cd-04125c92334b.png)

**Comment by gjost** [comment]
It would be nice to be able to just skip this step but the sqlite3 db is used by Celery and for tracking thumbnails so we do need it.

**Comment by gjost** [comment]
It's not real fix, but when you get to this step you can regenerate the virtualenv and it'll work:
```
cd /opt/ddr-local
sudo rm -Rf /opt/ddr-local/venv
sudo make install
sudo make syncdb
```

**Comment by gjost** [comment]
Fixed in cd666e2.
